@ecomplus/storefront-app/src/components/js/EcOrderInfo.js

```diff
@@ -144,6 +149,13 @@ export default {
         : {}
     },
 
+    isValidThru () {
+      const transactionMethod = this.transaction['banking_billet'] || this.transaction['account_deposit']
+      return transactionMethod && 
+        transactionMethod.valid_thru &&
+        Date.now() < new Date(transactionMethod.valid_thru).getTime()
+    },
```

Comment:
```suggestion
    validThru () {
      const transactionMethod = this.transaction['banking_billet'] || this.transaction['account_deposit']
      return transactionMethod && transactionMethod.valid_thru
    },
```

Reply:
Como são as coisas kkk, eu fiz dessa forma no início, mas ai pensei, vou seguir o padrão que é colocar os is..., ai voltei. Assim fica bem mais enxuto mesmo